src/components/BookInfo/BookDetails/BookDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent } from "react";
+import { TextContent } from "../../global/TextContent/TextContent";
+import { BookDetailsProps } from "./interfaces/BookDetailsProps";
+import styles from "./BookDetails.module.scss";
+import moment from "moment";
+import { IndustryIdentifier } from "../../BookFinder/BookList/interfaces/IndustryIdentifier";
+
+export const BookDetails: FunctionComponent<BookDetailsProps> = (props) => {
+  const renderIndustryIdentifiers = (
+    industryIdentifiers: IndustryIdentifier[]
+  ) => {
+    return industryIdentifiers
+      .filter(
+        (industryIdentifier: IndustryIdentifier) =>
+          industryIdentifier.type === "ISBN_10" ||
+          industryIdentifier.type === "ISBN_13"
+      )
+      .map((industryIdentifier: IndustryIdentifier) => {
+        return (
+          <React.Fragment key={industryIdentifier.identifier}>
+            <TextContent className={styles.label} fontSize={12}>
+              {industryIdentifier.type.replace("_", "-")}
+            </TextContent>
+            <TextContent fontSize={18}>
+              {industryIdentifier.identifier}
+            </TextContent>
+          </React.Fragment>
+        );
+      });
+  };
+
+  const renderOtherCategories = (categories: string[]) => {
+    categories.shift();
+    return categories.map((category: string) => {
+      return (
+        <TextContent key={category} className={styles.badge}>
+          {category}
+        </TextContent>
+      );
+    });
```

Comment:
Zakej koristiš .shift() ? Postoji li neki special reason?

Reply:
Plus, ako već trebaš taj prvi item iz array-a, renderiraj ga bez .map jer je singularni item. .map se uobičajeno koristi za multiple rendering, ovako izgleda weird a znamo da se rendera samo jedan item. Probaj ne koristiti .map za singular rendere. Not sure da li je ovo praksa, no na taj način pišemo reno i ja odkad sam krenul delati

Reply:
ova funkcija rendera sve osim prvog itema
na jednom drugom mjestu ispisem prvu kategoriju iz liste, a na ovom shiftam (izbacim prvi element iz postojeceg arraya) i renderam sve ostale. more ih bit vise pa zato map :D

Reply:
Aha, sori. Čitajući definiciju s googla me zbunilo, sada vidim što si htio napraviti..Ima smisla, very good :)